src/wallet/rpcwallet.cpp

```diff
@@ -416,12 +419,15 @@ UniValue sendtoaddress(const JSONRPCRequest& request)
             "                             transaction, just kept in your wallet.\n"
             "5. subtractfeefromamount  (boolean, optional, default=false) The fee will be deducted from the amount being sent.\n"
             "                             The recipient will receive less bitcoins than you enter in the amount field.\n"
+            "6. allowdirty             (boolean, optional, " + (GetBoolArg("-avoidreuse", false) ? "default=false" : "unavailable") + ") Allows spending from dirty addresses; addresses are considered\n"
```

Comment:
Why `unavailable`? Someone may just have enabled `-avoidreuse` but hasn't rescanned. Maybe keep a state somewhere if we have scanned with `-avoidreuse` up to the wallets bestblock.